src/instructlab/model/accelerated_train.py

```diff
@@ -32,8 +32,8 @@
 class SupportedTrainingStrategies(enum.Enum):
     """Available advanced training strategies"""
 
-    LAB_MULTIPHASE: str = "lab-multiphase"
```

Comment:
this is a violation of python enum rules, see https://typing.python.org/en/latest/spec/enums.html#defining-members

> Members defined within an enum class should not include explicit type annotations. Type checkers should infer a literal type for all members. A type checker should report an error if a type annotation is used for an enum member because this type will be incorrect and misleading to readers of the code: